prime-router/src/main/java/fhirengine/translation/hl7/structures/fhirinventory/group/ORM_O01_ORDER_DETAIL.java

```diff
@@ -0,0 +1,360 @@
+ * Software distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+ * specific language governing rights and limitations under the License.
+ *
+ * The Original Code is "[file_name]".  Description:
+ * "[one_line_description]"
+ *
+ * The Initial Developer of the Original Code is University Health Network. Copyright (C)
+ * 2012.  All Rights Reserved.
+ *
+ * Contributor(s): ______________________________________.
+ *
+ * Alternatively, the contents of this file may be used under the terms of the
+ * GNU General Public License (the  "GPL"), in which case the provisions of the GPL are
+ * applicable instead of those above.  If you wish to allow use of your version of this
+ * file only under the terms of the GPL and not to allow others to use your version
+ * of this file under the MPL, indicate your decision by deleting  the provisions above
+ * and replace  them with the notice and other provisions required by the GPL License.
+ * If you do not delete the provisions above, a recipient may use your version of
+ * this file under either the MPL or the GPL.
+ *
+ */
+
+package fhirengine.translation.hl7.structures.fhirinventory.group;
+
+import ca.uhn.hl7v2.HL7Exception;
+import ca.uhn.hl7v2.model.v27.segment.*;
+import ca.uhn.hl7v2.model.*;
+import ca.uhn.hl7v2.parser.ModelClassFactory;
+import fhirengine.translation.hl7.structures.fhirinventory.segment.NTE;
+import fhirengine.translation.hl7.structures.fhirinventory.segment.OBR;
+
+/**
+ * <p>Represents a ORM_O01_ORDER_DETAIL group structure (a Group object).
+ * A Group is an ordered collection of message segments that can repeat together or be optionally in/excluded together.
+ * This Group contains the following elements:
+ * </p>
+ * <ul>
+ * <li>1: OBR (Observation Request) <b>  choice</b></li>
+ * <li>2: RQD (Requisition Detail) <b>  choice</b></li>
+ * <li>3: RQ1 (Requisition Detail-1) <b>  choice</b></li>
+ * <li>4: RXO (Pharmacy/Treatment Order) <b>  choice</b></li>
+ * <li>5: ODS (Dietary Orders, Supplements, and Preferences) <b>  choice</b></li>
+ * <li>6: ODT (Diet Tray Instructions) <b>  choice</b></li>
+ * <li>7: NTE (Notes and Comments) <b>optional repeating </b></li>
+ * <li>8: CTD (Contact Data) <b>optional  </b></li>
+ * <li>9: DG1 (Diagnosis) <b>optional repeating </b></li>
+ * <li>10: ORM_O01_OBSERVATION (a Group object) <b>optional repeating </b></li>
+ * </ul>
+ * <p>
+ * Note that this structure has "choice" elements. This means that one of the
+ * structures above marked as "choice" must be present, but no more than one.
+ * </p>
+ */
+//@SuppressWarnings("unused")
+public class ORM_O01_ORDER_DETAIL extends AbstractGroup {
+
+  /**
+   * Creates a new ORM_O01_ORDER_DETAIL group
+   */
+  public ORM_O01_ORDER_DETAIL(Group parent, ModelClassFactory factory) {
+    super(parent, factory);
+    init(factory);
+  }
+
+  private void init(ModelClassFactory factory) {
```

Comment:
Could we add a comment here indicating that caristix says there is an `ORDER_DETAIL_SEGMENT` which doesn't actually get modeled?

Reply:
Good idea, [added comment](https://github.com/CDCgov/prime-reportstream/pull/16299/commits/f129f2c62d0e9ca370d91468e08c3e13d6b7bc4c).